### DISCUSSION TITLE
[Feature Request] Enable row-level filters

### DISCUSSION THREAD

**Discussion by lucaswhornsby** [grafana/grafana#58489] [Dashboards, Panels & Visualizations]
Currently, it is only possible to apply global filters to the whole dashboard. However, the ability to apply filters to individual rows would allow more granular pivoting of our data that might only be relevant to a specific set of panels and not the remainder of the dashboard.

As such, we would like to be able to configure and apply filters to individual rows on a dashboard. For example, I would like to be able to apply a filter (by whatever dimensions I customize) to a particular set of panels like a pie chart, bar graph, and line graph that comprise a particular row in my dashboard, and these row-level filters would be displayed below the row title instead of at the top of the dashboard.

**Comment by StevenGrzybowski** [comment]
I would second this notion - for example, when trying to dashboard out a large number oof instances of service, it is much nicer if you could design a set of panels within a row per instance, then repeat the row by instance with each set of panels being filtered to the single value of the instance

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
